providers/configcat/go.sum

```diff
@@ -1106,8 +1109,6 @@ github.com/yuin/goldmark v1.4.1/go.mod h1:mwnBkeHKe2W/ZEtQ+71ViKU8L12m81fl3OWwC1
 github.com/yuin/goldmark v1.4.13/go.mod h1:6yULJ656Px+3vBD8DxQVa3kxgyrAnzto9xy5taEt/CY=
 github.com/zeebo/assert v1.3.0/go.mod h1:Pq9JiuJQpG8JLJdtkwrJESF0Foym2/D9XMU5ciN/wJ0=
 github.com/zeebo/xxh3 v1.0.2/go.mod h1:5NWz9Sef7zIDm2JHfFlcQvNekmcEl9ekUZQQKCYaDcA=
-go.buf.build/open-feature/flagd-connect/open-feature/flagd v1.1.3/go.mod h1:ZaEOUjTu/oxTFHuwNjF2788REepgWYPsU4Ty6lPIMzc=
-go.buf.build/open-feature/flagd-connect/open-feature/flagd v1.1.4/go.mod h1:ZaEOUjTu/oxTFHuwNjF2788REepgWYPsU4Ty6lPIMzc=
```

Comment:
@Kavindu-Dodan this is the "bad" dep. Its ssem to be coming in via the sdk, but I can't understand why (the SDK version seems new enough that it shouldn't need this).